android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/EngineDelegateMiddleware.kt

```diff
@@ -44,8 +44,8 @@ internal class EngineDelegateMiddleware(
             is EngineAction.ExitFullScreenModeAction -> exitFullScreen(context.store, action)
             is EngineAction.SaveToPdfAction -> saveToPdf(context.store, action)
             is EngineAction.PrintContentAction -> printContent(context.store, action)
-            is TranslationsAction.TranslateAction -> translate(context.store, action)
-            is TranslationsAction.TranslateRestoreAction -> translateRestoreOriginal(context.store, action)
+            is TranslationsAction.TranslateAction -> translate(context.store, next, action)
+            is TranslationsAction.TranslateRestoreAction -> translateRestoreOriginal(context.store, next, action)
```

Comment:
Could we reorder these `TranslationsAction` to be after the `EngineAction`? Just nice to keep all the `EngineAction` altogether.

Reply:
Sure, updated!